src/main/java/org/folio/service/CustomFieldsServiceImpl.java

```diff
@@ -30,8 +31,23 @@
 
 import javax.annotation.Nullable;
 
+import com.google.common.collect.Sets;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
 import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang.WordUtils;
```

Comment:
is this class available in org.apache.commons.lang3?
if yes, please replace

Reply:
It is deprecated in `org.apache.commons.lang3` and moved to the `org.apache.commons.commons-text` that is not in dependencies of the project